test/flutter/templates.test.ts

```diff
@@ -9,13 +9,13 @@ describe('Flutter code templates', () => {
     it('generates pubspec with project and org', () => {
       const template = pubspecOptions('fixture-project', 'fixture-org');
       expect(template).toMatchInlineSnapshot(`
-      "sentry:
-        upload_debug_symbols: true
-        upload_source_maps: true
-        project: fixture-project
-        org: fixture-org
-      "
-      `);
+              "sentry:
```

Comment:
are these new spacings from auto formatting?

Reply:
honestly not sure. These are generated automatically when running `jest -u` to update the inline snapshots. I think it's fine though because the actual indentation is relative to the first `"`.